### ISSUE TITLE
Support for small caps

### ISSUE THREAD

**Issue by vgmoose** [mwilliamson/mammoth.js#123]
In Word there is a concept of [small caps](https://en.wikipedia.org/wiki/Small_caps), which are capital letters where the lowercase letters are still capitalized, but just not as big as the actual capital letters.

One use-case where this property has significance is while making citations in footnotes.

Here's an example doc.x that uses the property: [Small Caps Example.docx](https://github.com/mwilliamson/mammoth.js/files/1157598/Plug.in.test.docx)

Which looks like:
![image1](https://user-images.githubusercontent.com/2467473/28344557-387ac2de-6bf2-11e7-908b-cbc38f175c6a.jpeg)

Fortunately, the same concept seems to exist in HTML using: 
```
.small {
    font-variant: small-caps;
}
```
which creates the same effect.

Would this be something that is easily incorporated into mammoth?

**Comment by mwilliamson** [comment]
Small caps doesn't have a natural HTML element to map to, so the solution that makes most sense to me would be to add support for adding style mappings for small caps, in the same that underline and similar are currently supported. Otherwise, you can apply small caps through a Word style, and map that Word style to an appropriate CSS class.

**Comment by vgmoose** [comment]
Thanks, I think I understand the style-mapping thing now, and created a pull request at #124 

My use case for this is with the wordpress .docx mammoth plugin, so I would prefer it to pull out small caps without having to create a Word style. In the PR I tried to look at how w:u was implemented, but parts of it may be off.

Let me know if I'm missing something about how the mapping works, and I can edit the pull request as well.
